### ISSUE TITLE
Feature request: phone requires + for US

### ISSUE THREAD

**Issue by gburchers** [larrylisky/Dashphone#5]
US phone number required + at beginning.  More conventional to have no + and potentially auto-add the +1 to any US phone number.

**Comment by larrylisky** [comment]
Agreed.  There is a more "perfect" implementation possible.  Target it for next milestone (v0.10).
